test/renameKeyWith.js

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+import { concat } from 'ramda';
+
+import * as RA from '../src';
+
+describe('renameKeyWith', function () {
+  it('should rename object keys', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', { A: 1 }), {
+      aA: 1,
+    });
+  });
+
+  it('should not change other key-value pairs', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', { A: 1, B: 2 }), {
+      aA: 1,
+      B: 2,
+    });
+  });
+
```

Comment:
Let's add test for a case when we want to rename non-existing key

Reply:
Done